src/main/java/org/opentripplanner/routing/api/request/refactor/preference/StreetPreferences.java

```diff
@@ -0,0 +1,175 @@
+package org.opentripplanner.routing.api.request.refactor.preference;
+
+import java.io.Serializable;
+import java.time.Duration;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.Map;
+import org.opentripplanner.routing.algorithm.filterchain.ItineraryListFilter;
+import org.opentripplanner.routing.api.request.StreetMode;
+import org.opentripplanner.routing.impl.DurationComparator;
+import org.opentripplanner.routing.impl.PathComparator;
+import org.opentripplanner.routing.spt.GraphPath;
+
+// Direct street search
+public class StreetPreferences implements Cloneable, Serializable {
```

Comment:
We need to look at this class again, above the class it say this is for street search, but why do this class then have fields for access/egress. Also some of the names repeat the context.

@hannesj and @leonardehrenfried We can fix this later if it does not break anything now.